docs/root/intro/version_history.rst

```diff
@@ -28,7 +29,7 @@ Version history
   health check/weight/metadata updates within the given duration.
 * config: regex validation added to limit to a maximum of 1024 characters.
 * config: v1 disabled by default. v1 support remains available until October via flipping --v2-config-only=false.
-* config: v1 disabled by default. v1 support remains available until October via setting :option:`--allow-deprecated-v1-api`.
+* config: v1 disabled by default. v1 support remains available until October via deprecated flag --allow-deprecated-v1-api.
```

Comment:
I don't think we should modify earlier release notes, this is a bit like rewriting history. Ideally once we release, they are immutable.

Reply:
Unfortunately I had to - the :option: tag was breaking the docs build now that the flag didn't exist.
I figured as long as I was removing the tagging I should explain why.

Reply:
So be it.